src/middleware/jwt.middleware.ts

```diff
@@ -0,0 +1,17 @@
+import * as jwt from 'jsonwebtoken';
+
+export function checkToken(req, res, next) {
+    const token = req.headers['x-access-token'];
+
+    if (!token) {
+        return res.status(403).send({ success: false, message: 'Forbidden Error' });
```

Comment:
Please replace this line with 12, since we should send 403 error if token is invalid, not absent